### PR TITLE
Use ATK_STATE_SELECTED and ATK_STATE_FOCUSED appropriately for highli…

### PR DESCRIPTION
**Pull request by TheMuso**
Allows the user to get feedback from Orca when additional icons are selected or deselected.

### CODE REVIEW

File: libcaja-private/caja-icon-container.c [hunk 1]
```diff
@@ -856,10 +856,10 @@ clear_keyboard_focus (CajaIconContainer *container)
 }
 
 static void inline
-emit_atk_focus_tracker_notify (CajaIcon *icon)
+emit_atk_focus_tracker_notify (CajaIcon *icon, gboolean focused)
```
Comment (cwendling) [inline libcaja-private/caja-icon-container.c]: This function could use some renaming then I guess

**Comment by cwendling** [comment]
Looks good, and it indeed fixes Orca not announcing anything while extending a selection for me. :+1:

**Comment by TheMuso** [comment]
On Fri, Jan 27, 2017 at 07:44:29PM AEDT, Colomban Wendling wrote:
>  static void inline
> -emit_atk_focus_tracker_notify (CajaIcon *icon)
> +emit_atk_focus_tracker_notify (CajaIcon *icon, gboolean focused)
> 
> This function could use some renaming then I guess

Yeah good point.

Pushed a commit to do just that.

Luke

**Comment by monsta** [comment]
Nice, it should fix that deprecation warning about ```atk_focus_tracker_notify```. Looks ok to me.

**Comment by monsta** [comment]
The last two commits could be made simpler though, the emit function is always called with ```TRUE``` as second argument, so it doesn't seem to be needed.

**Comment by monsta** [comment]
It seems to me that https://github.com/mate-desktop/caja/issues/281 might be fixed by this PR. Can you guys confirm it please?

**Comment by alexarnaud** [comment]
@monsta it seems it PR resolved effectively the bug #281.

**Comment by monsta** [comment]
Thanks, it's closed now.

**Pull request merged**